stories/windows_rdp_artifacts_and_defense_evasion.yml

```diff
@@ -0,0 +1,19 @@
+name: Windows RDP Artifacts and Defense Evasion
+id: 22c5dc79-b418-4933-b8ba-9431dfc436a8
+version: 2
+date: '2025-07-30'
+author: Teoderick Contreras, Splunk
+status: production
+description: This detection identifies the use of Remote Desktop Protocol (RDP) on a Windows system followed by actions consistent with artifact cleanup or defense evasion. When a user initiates an RDP session using the native client (mstsc.exe), Windows generates several artifacts, including Default.rdp in the user's Documents folder and bitmap cache files (*.bmc, cache*.bin) under the Terminal Server Client cache directory. These files can be valuable for forensic analysis, as they indicate remote access activity and sometimes reveal details about the accessed system's graphical environment.
```

Comment:
@t-contreras - seems like there is some discrepancy with the description. Can we start with : This analytic story ?